pgml-dashboard/static/css/scss/abstracts/variables.scss

```diff
@@ -120,6 +120,7 @@ $hp-white: #{$gray-200};
 // Background Colors
 $bg-white: #F8FAFC;
 $bg-black: #0D0D0E;
+$bg-black-1: #17181A;
```

Comment:
We shouldn't need to add any colors.  this is gray-800. If we need this variable to it like `$bg-black-1: #{$gray-800};` or just use gray-800 in the css you are trying to use this for.